pkg/openstack/loadbalancer.go

```diff
@@ -606,6 +607,22 @@ func (lbaas *LbaasV2) getLoadBalancerLegacyName(_ context.Context, _ string, ser
 	return cloudprovider.DefaultLoadBalancerName(service)
 }
 
+// Returns a list of custom load balancer listener tags for the specified  service.
+func (lbaas *LbaasV2) getCustomLoadBalancerTags(service *corev1.Service) []string {
+	var listenerTags []string
+	annotationVal := getStringFromServiceAnnotation(service, ServiceAnnotationLoadBalancerCustomTags, "")
+
+	listOfTags := strings.Split(annotationVal, ",")
+
+	for _, tags := range listOfTags {
+		tags = strings.TrimSpace(tags)
+
+		listenerTags = append(listenerTags, tags)
+	}
+
+	return listenerTags
```

Comment:
Are there just listener tags? According to your [description](https://github.com/kubernetes/cloud-provider-openstack/pull/2439/files#diff-7f585750e84ff958bbc414045cc178a9fa7f91c86be09356ff549142e1430a24R242). It should also contain loadbalancer tag and pool tag.